zjit/src/hir_type/mod.rs

```diff
@@ -143,6 +143,10 @@ fn is_range_exact(val: VALUE) -> bool {
     val.class_of() == unsafe { rb_cRange }
 }
 
+fn is_set_exact(val: VALUE) -> bool {
```

Comment:
Please inline this; the other ones are only outlined because of the prism GC-hiding weirdness